release/kata-deploy-binaries.sh

```diff
@@ -128,12 +128,16 @@ install_kata_components() {
 		libexecdir="libexec"
 		if [ "$p" == "shim" ]; then
 			libexecdir="/${destdir}/${prefix}/libexec/"
+			make PREFIX="${prefix}" \
+				DESTDIR="${destdir}" \
+				LIBEXECDIR="${libexecdir}" \
```

Comment:
Nit: `libexecdir` is only used in this block now, so you could remove the variable from above the `if`.

Reply:
Or to avoid duplicating make call you can use:

```
makeVars=(PREFIX="${prefix}" DESTDIR="${destdir}")

if [ "$p" == "shim" ]; then
	libexecdir="/${destdir}/${prefix}/libexec/"
	makeVars+=(LIBEXECDIR="${libexecdir}")
fi

make install ${makeVars[@]}
```